lib/units.scss

```diff
@@ -10,3 +10,10 @@ $mmf-height-label: 1.1em !default;
 // animation
 $mmf-transition--time: 0.1s;
 $mmf-transition: $mmf-transition--time linear;
+
+// text & line styles
+$mmf-line-height: 140.62%;
+$mmf-letter-spacing: 0.05em;
+$mmf-font-normal: 14px;
```

Comment:
Dont use absolute font-sizes. At least try using root-relative values (where 16px are default). e.g `$mmf-font-normal: 14/16 + rem`. This enables font-scaling on a global level (increasing font-size on body)